go/libraries/doltcore/sqle/dolt_diff_table_function.go

```diff
@@ -99,16 +103,27 @@ func (dtf *DiffTableFunction) WithExpressions(expression ...sql.Expression) (sql
 		}
 	}
 
-	dtf.fromCommitExpr = expression[0]
-	dtf.toCommitExpr = expression[1]
-	dtf.tableNameExpr = expression[2]
+	if strings.Contains(expression[0].String(), "..") {
```

Comment:
Seems like this will throw an error without the `len(expression)` check if no expressions are passed in? Would be good to add a quick test assertion for `select * from dolt_diff()` returning an invalid argument error.

Reply:
I added that check/test in my `dolt_diff_summary` PR [here](https://github.com/dolthub/dolt/pull/4646/files#diff-51ab7d0a893924f00c2f31c46b1b61a64af3f1ea9a7a9ddda7cdf885d61094e1R4506). Will merge that into this one merge I merge to main